cirq-core/cirq/ops/gateset.py

```diff
@@ -394,8 +397,6 @@ def _validate_operation(self, op: raw_types.Operation) -> bool:
                 lambda q: cast(circuit_operation.CircuitOperation, op).qubit_map.get(q, q)
             )
             return self.validate(op_circuit)
-        elif isinstance(op, global_phase_op.GlobalPhaseOperation):
-            return self._accept_global_phase_op
```

Comment:
This should be removed when we actually remove`GlobalPhaseOperation` after the deprecation cycle to mitigate immediate breakages in user code?

Reply:
Can be removed now because GlobalPhaseOp.gate exists, so it'll go through the `gate` path.